WIP: Add other browser devtools links (#309)
webrtc-for-the-curious/webrtc-for-the-curious: content/docs/09-debugging.md

[Wireshark](https://www.wireshark.org) is a widely-used network protocol analyzer.

> ### Browsers' developer tools

Reviewer: ```suggestion
### WebRTC browser tools
```

> Browsers come with built-in pages where you can closer inspect the connections you make. Chromium (Chrome): [`chrome://webrtc-internals`](chrome://webrtc-internals), [`chrome://webrtc-logs`](chrome://webrtc-logs); Gecko (Firefox): [`about:webrtc`](about:webrtc).

Reviewer: I'd prefer this to be more in text form, as full sentences.
```suggestion
Browsers come with built-in tools that you can use to inspect the connections you make. Chrome has [`chrome://webrtc-internals`](chrome://webrtc-internals) and [`chrome://webrtc-logs`](chrome://webrtc-logs). Firefox has [`about:webrtc`](about:webrtc).
```
